### PR TITLE
these creatures should have no loot

### PR DESCRIPTION
**Pull request by Flashaholic88**
## CHANGES PROPOSED:
-  removed lootid from creatures that dont have loot at all

## Target branch(es):
- [x] Master

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Kitzunu** [comment]
We got some trouble here:
1. Remove loot.sql
2. Add that into the pending sql.
3. What are these changes based on?

**Comment by Kitzunu** [comment]
1. Please fix the DBErrors caused by this
2. Pls tell us what the changes are based on, we cant approve if we dont know :/

**Comment by Flashaholic88** [comment]
these changes are based of wowhead.
i just wondering why i can get loot from totems, snakes and such

File: data/sql/updates/pending_db_world/rev_1588410722246016400.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588410722246016400');
+
+UPDATE `creature_template` SET `lootid` = '0' WHERE (`entry` = '2992', '7484', '3680', '17079', '17080');
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1588410722246016400.sql:3]: `lootid` is a number, not a string, same for `entry` values

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1588410722246016400.sql:3]: + the entry change makes no sense.
go back to WHERE `entry` IN (nnnn)

**Comment by Kitzunu** [comment]
Okay I will try to explain this how to do it the right way, because it was partly right the first thing you did :)

The correct way to do the SQL is this way
```sql
UPDATE `creature_template` SET `lootid` = 0 WHERE `entry` IN (2992, 7484, 3680, 17079, 17080); 
```
Which is close to what you had at first.

The wrongs that you did were that you were using `''` around non-string numbers. There is close to nowhere in the database where you would need to use it around numbers.

The second errors are these DBErrors that you need to fix as well
```
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 2992 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 3680 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 7484 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 17079 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 17080 isn't creature entry and not referenced from loot, and thus useless.
```

So basically how to proceed is to revert your previous 2 commits to the state it was then, delete the `''` around the numbers and then attempt to fix the DBErrors I mentioned above.
If you need any help with it, feel free to comment and I am more than happy to assist you :)

**Comment by Kitzunu** [comment]
Okay so I will assist you in how to fix the DBErrors:
```
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 2992 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 3680 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 7484 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 17079 isn't creature entry and not referenced from loot, and thus useless.
2020-05-02 12:19:09 Table 'creature_loot_template' Entry 17080 isn't creature entry and not referenced from loot, and thus useless.
```
Basically what this says is that creature_loot_template has entries, however, they are not used and thus useless.

Since you removed these from `lootid` in creature_template you also have to remove these entries in creature_loot_template since they are not used by any other creature.

**Comment by merdlin** [comment]
why is there normal loot in these creatures?

**Comment by merdlin** [comment]
Tested, it really works.

**Comment by Kitzunu** [comment]
Ty for the pr
